tfe/resource_tfe_workspace_test.go

```diff
@@ -249,7 +249,7 @@ func TestAccTFEWorkspace_update(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						"tfe_workspace.foobar", "file_triggers_enabled", "true"),
 					resource.TestCheckResourceAttr(
-						"tfe_workspace.foobar", "operations", "false"),
+						"tfe_workspace.foobar", "operations", "true"),
```

Comment:
Why did you change this? It seems to be unrelated to your changes right?

Reply:
If you check out master branch this dest is failing without any changes. This makes it work as a baseline.

Reply:
I could skip this test and create new one just for this feature instead of relying on the same test that checks now 3 different things. Let me know if that's proffered approach I can change that.

Reply:
@svanharmelen Latest commit is removing those changes. I opted in for separate dedicated test for workspace updates.